deployment/ansible/raster-foundry.yml

```diff
@@ -18,3 +18,7 @@
     - name: Set Environment variable for AWS profile
       lineinfile: dest=/etc/environment regexp=^AWS_PROFILE line="AWS_PROFILE={{aws_profile}}"
       when: aws_profile is defined
+
+    - name: Set Environment variable for SWF Domain
+      lineinfile: dest=/etc/environment regexp=^LAMBDA_ALIAS line="SWF_DOMAIN={{swf_domain}}"
```

Comment:
I think you want `regexp` to look for `SWF_DOMAIN`. It may also be useful to prefix this with `AWS_` so that it is clear what is AWS specific and what is not.